tm2/pkg/crypto/keys/client/addpkg.go

```diff
@@ -6,8 +6,10 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"path/filepath"
 
 	gno "github.com/gnolang/gno/gnovm/pkg/gnolang"
+	"github.com/gnolang/gno/gnovm/pkg/gnomod"
```

Comment:
This one is forbidden.

Maybe we should wait for TM2 split PR started by @piux2, then you'll have the `AddPkg` not in `tm2` anymore, but in `gno.land`.

@harry-hov, what about trying to help @piux2 finishing his PR first?

Reply:
the split is finalised so the problematic import can be fixed, please merge master :)